framework/src/timesteppers/IterationAdaptiveDT.C

```diff
@@ -232,7 +243,10 @@ IterationAdaptiveDT::computeDT()
   else if (_sync_last_step)
```

Comment:
I've been stuck on what's going on here @loganharbour for the recover test. The reset timestep, here given by `_post_function_sync_dt`, is not applied during a restart, despite being applicable: With the non-restarted test, `_sync_last_step=true`, so `_post_function_sync_dt` is used. During the restart, despite `_sync_last_step` being a `declareRestartableData`, it does not retain the `true` value during the recover. I'm a bit lost on why this doesn't work?

Reply:
@dschwen ?